hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/raft/StoreSnapshotFile.java

```diff
@@ -48,7 +47,7 @@
 
     private static final Logger LOG = Log.logger(StoreSnapshotFile.class);
 
-    private static final String SNAPSHOT_DIR = "snapshot";
+    public static final String SNAPSHOT_DIR = "snapshot";
```

Comment:
protected

Reply:
AbstractBackendStoreProvider has used it.